src/background/background.js

```diff
@@ -59,12 +59,32 @@ function getData() {
     return data
 }
 
-function addToken(tokenId, tokenSymbol, networkByte) {
+function updateVersion() {
```

Comment:
It seems this function just do something related with Token, not version. Please rename the function name.

Reply:
And this function could move to `TokenRecord.vue` or `Home.vue`.